src/translib/common_app.go

```diff
@@ -258,7 +258,7 @@ func (app *CommonApp) translateCRUDCommon(d *db.DB, opcode int) ([]db.WatchKeys,
 	log.Info("translateCRUDCommon:path =", app.pathInfo.Path)
 
 	// translate yang to db
-	result, err := transformer.XlateToDb(app.pathInfo.Path, opcode, d, (*app).ygotRoot, (*app).ygotTarget, txCache)
+	result, err := transformer.XlateToDb(app.pathInfo.Path, opcode, d, (*app).ygotRoot, (*app).ygotTarget, (*app).body, txCache)
```

Comment:
@ranjinidn  -- please review changes to this file for RBAC.

Reply:
This change is already in dell_sonic. If you rebase to dell_sonic, you should get only your changes added in this branch.